src/components/CustomTooltip.tsx

```diff
@@ -0,0 +1,84 @@
+/** @jsx jsx */ import { jsx, css } from '@emotion/core';
```

Comment:
if you have control over your babel config/build process it might be worth using the babel plugin (https://emotion.sh/docs/css-prop) instead of using pragma. Downside of pragma is you have to remember to use it in any emotion files.

Reply:
according to the emotion docs it's not configurable :( https://emotion.sh/docs/css-prop#babel-preset . I'm also hoping to switch this to a next app once this is fixed https://github.com/vercel/next.js/issues/12079 so could be moot anyway :)